vike/node/plugin/plugins/importUserCode/v1-design/getVikeConfig/configDefinitionsBuiltIn.ts

```diff
@@ -206,6 +207,9 @@ const configDefinitionsBuiltInGlobal: Record<ConfigNameGlobal, ConfigDefinitionI
   onBeforeRoute: {
     env: { server: true, client: 'if-client-routing', eager: true }
   },
+  onBoot: {
+    env: { server: true, client: true, eager: true }
```

Comment:
~~I'm not sure, but I don't think we need `eager`.~~

Actually, we need `eager` if we make `onBoot()` non-global and cumulative (see my sibling comment).